src/participant/participant.ts

```diff
@@ -67,24 +74,61 @@ export function getRunnableContentFromString(text: string): string {
 
 export default class ParticipantController {
   _participant?: vscode.ChatParticipant;
+  _context: vscode.ExtensionContext;
   _connectionController: ConnectionController;
   _storageController: StorageController;
   _chatMetadataStore: ChatMetadataStore;
+  _docsChatbotAIService: DocsChatbotAIService;
   _telemetryService: TelemetryService;
 
   constructor({
+    context,
     connectionController,
     storageController,
     telemetryService,
   }: {
+    context: vscode.ExtensionContext;
     connectionController: ConnectionController;
     storageController: StorageController;
     telemetryService: TelemetryService;
   }) {
+    this._context = context;
     this._connectionController = connectionController;
     this._storageController = storageController;
     this._chatMetadataStore = new ChatMetadataStore();
     this._telemetryService = telemetryService;
+
+    const docsChatbotBaseUri = this._readDocsChatbotBaseUri();
+    this._docsChatbotAIService = new DocsChatbotAIService(docsChatbotBaseUri);
+  }
+
+  // To integrate with the MongoDB documentation chatbot,
+  // set the MONGODB_DOCS_CHATBOT_BASE_URI environment variable when running the extension from a branch.
+  // This variable is automatically injected during the .vsix build process via GitHub Actions.
+  private _readDocsChatbotBaseUri(): string | undefined {
+    config({ path: path.join(this._context.extensionPath, '.env') });
```

Comment:
Since `_context` is only used in this function, which is used in the constructor, how does it sound if we avoid attaching it to the class and instead pass it through as an argument to the function?